pkgs/applications/networking/sieve-connect/default.nix

```diff
@@ -0,0 +1,36 @@
+{ stdenv, fetchurl, makeWrapper, perl, perlPackages }: stdenv.mkDerivation rec {
+  name = "sieve-connect-${version}";
+  version = "0.89";
+
+  src = fetchurl {
```

Comment:
Please use `fetchFromGitHub` instead of `fetchurl` because it relies on the hash of the contents of the repository instead of the hash of a tarball, and it would allow us to fix all GitHub src fetchers, if GitHub changes something about how it works.